django/contrib/auth/hashers.py

```diff
@@ -704,3 +704,70 @@ def safe_summary(self, encoded):
 
     def harden_runtime(self, password, encoded):
         pass
+
+
+class ScryptPasswordHasher(BasePasswordHasher):
```

Comment:
I wouldn't sort this at the bottom of the file below all the insecure hashers.

Reply:
I moved it up to the section with the secure hashers, just under the Bcrypt password hasher.